compiler/circle-partitioner/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+file(GLOB_RECURSE SOURCES "src/*.cpp")
+
+add_executable(circle_partitioner "${SOURCES}")
+target_link_libraries(circle_partitioner safemain)
+target_link_libraries(circle_partitioner arser)
+target_link_libraries(circle_partitioner vconone)
+
+install(TARGETS circle_partitioner DESTINATION bin)
```

Comment:
@seanshpark Seems like the name isn't consistent with others. Maybe it can be `circle-partitioner`.
```bash
$ ls              
circle2circle                  model2nnpkg.sh          one-import-tf      __pycache__     tflchef-file
circle_partitioner             one-build               one-import-tflite  python          tflchef-reverse
circle-quantizer               one-build.template.cfg  one-optimize       rawdata2hdf5    tflite2circle
generate_bcq_metadata          one-codegen             one-pack           record-minmax   utils.py
generate_bcq_metadata.py       one-import              one-prepare-venv   tf2nnpkg        venv
generate_bcq_output_arrays     one-import-bcq          one-quantize       tf2tfliteV2.py
generate_bcq_output_arrays.py  one-import-onnx         one-version        tflchef

```

Reply:
> Seems like the name isn't consistent with others.

Yes, seems so.... Please post a PR to rename if you like :)